Fix traceback from prune() call discovered after `pulp/pulpcore#5934` addressed.
